plsync/planetlab/model.py

```diff
@@ -589,6 +608,29 @@ def __init__(self, **kwargs):
 
         super(Slice, self).__init__(**kwargs)
 
+    def dnsname(self):
+        """Returns the slice name in DNS form, e.g. group_name to name.group"""
+        fields = self['name'].split('_')
+        # When a name has multiple '_', rejoin all parts after the group name.
```

Comment:
Much more concise!  4 lines of code reduced to a single, and as it turns out, more comprehensive line.  LGTM.